README.md

```diff
@@ -50,6 +50,22 @@ Datashader is not currently provided on pip/PyPI, to avoid broken or
 low-performance installations that come from not keeping track of
 C/C++binary dependencies such as LLVM (required by Numba).
 
+One way to easily install `datashader` and related GIS and visualization tools is to install `earth-env` ([source code](http://github.com/ContinuumIO/earth-env)) with one a command like the following for Python 3.6 (change the `-36` part for Python versions 2.7, 3.4, 3.5, or 3.6):
+
```

Comment:
earth-env is a private repo, so it can't be referenced here.  I'm not sure why earth-env has different environment specifications for different Python versions, but I don't see any differences between the files currently, and we would certainly strive to keep them the same.  With both those points in mind, I've put one environment.yml (envt name "ds") into datashader/examples/, and I suggest that you just use and discuss that one here.